tfexec/terraform.go

```diff
@@ -35,10 +35,8 @@ func NewTerraform(workingDir string, execPath string) (*Terraform, error) {
 	}
 
 	if execPath == "" {
-		execPath, err = FindTerraform()
-		if err != nil {
-			return nil, &ErrNoSuitableBinary{err: err}
-		}
+		err := fmt.Errorf("NewTerraform: please supply the path to a Terraform executable using execPath, e.g. using the tfinstall package.")
+		return nil, &ErrNoSuitableBinary{err: err}
```

Comment:
It may be useful to do this as `fmt.Errorf("...: %w", &ErrNoSuitableBinary{})` because it will force callers in to the `.Is` / `.As` consumption model.

Reply:
The error doesn't need to be wrapped for `.As()` to work - I've added a unit test which tests this.